src/pages/item/item.html

```diff
@@ -31,6 +31,6 @@
       </ion-item>
 		</ion-list>
 
-		<button ion-button (click)="onSave(itemForm)" type="submit" block>Save</button>
+		<button ion-button (click)="onSave(itemForm)" type="submit" block [disabled]="!selectedBrand || !selectedModel || !selectedCategory">Save</button>
```

Comment:
Technically they shouldn't contain invalid values since it gets the value from a dropdown (existing brands, models or categories). I see what you're saying though, it looks a bit ugly. 

Can you gray out fields with HTML5 form validation though? It can definitely be done with Angular 2 form validation, but it is a more complex implementation.